collector/netframework_clrjit.go

```diff
@@ -0,0 +1,119 @@
+// returns data points from Win32_PerfRawData_NETFramework_NETCLRJit
+// <add link to documentation here> - Win32_PerfRawData_NETFramework_NETCLRJit class
+package collector
+
+import (
+	"log"
+
+	"github.com/StackExchange/wmi"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	Factories["netframework_clrjit"] = NewNETFramework_NETCLRJitCollector
+}
+
+// A NETFramework_NETCLRJitCollector is a Prometheus collector for WMI Win32_PerfRawData_NETFramework_NETCLRJit metrics
+type NETFramework_NETCLRJitCollector struct {
+	NumberofMethodsJitted      *prometheus.Desc
+	TimeinJit                  *prometheus.Desc
+	StandardJitFailures        *prometheus.Desc
+	TotalNumberofILBytesJitted *prometheus.Desc
+}
+
+// NewNETFramework_NETCLRJitCollector ...
+func NewNETFramework_NETCLRJitCollector() (Collector, error) {
+	const subsystem = "netframework_clrjit"
+	return &NETFramework_NETCLRJitCollector{
+		NumberofMethodsJitted: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "jit_methods_total"),
+			"Displays the total number of methods JIT-compiled since the application started. This counter does not include pre-JIT-compiled methods.",
+			[]string{"process"},
+			nil,
+		),
+		TimeinJit: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "jit_timein"),
```

Comment:
Should have a unit, so `jit_time_seconds_total`?

Reply:
I need to double check what this one represents, I though it was the time spent during the last window and not the total time spent since the beginning of the app.